.travis.yml

```diff
@@ -179,7 +179,7 @@ jobs:
       install:
         - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release' '-DCMAKE_CXX_COMPILER=g++-5'
         - cmake --build build -- -j4
-      script: (cd build; ctest -V -L CORE)
+      script: (cd build; ctest -V -L CORE -j2)
```

Comment:
Looks like `ctest` is mixing an output of tests. I also do not know what happens when some test fail? Does it stop or it continues and exits with an error at the end?

Reply:
Good spot on the interleaving, thanks. My understanding is that ctests will stop and report the error after all running tests complete (i.e. it won't start running new tests after the failure, but in-flight tests will complete). I'm looking into how to deal with the interleaving question now and if possible, will update this PR with a fix for review before merging.

Reply:
one more thing (Maybe for different PR) cmake osx build .11 do not have ccache

Reply:
Ah yes, thats interesting. I'll look at getting ccache going in another PR. It was also pointed out to me recently that the ccache stats for cbmc travis builds don't look as good (i.e. more cache misses) than perhaps we might have expected, so I'll take a look at that in the same PR.

Reply:
I haven't found a way to make ctest group it's output more neatly, but it does prefix each output line with a number, so that you can still disambiguate which lines relate to which test directory. While it's not perfect, I think the time saving is probably worth it. I'll be happy to revert this though if anyone finds it too onerous to when debugging any failing tests.